docroot/modules/custom/sitenow_migrate/src/EventSubscriber/MigratePostImportEvent.php

```diff
@@ -128,6 +128,12 @@ public function onMigratePostImport(MigrateImportEvent $event) {
         $this->logger->notice($this->t('Checking for possible broken links'));
         $candidates = $this->checkForPossibleLinkBreaks();
         $this->updateInternalLinks($candidates);
+        break;
+
+      case 'd7_writinguniversity_blog_categories':
+        $this->addWritingUniversityCategories($migration);
+        break;
```

Comment:
FWIW, I think we are going to need to look at a different way of implementing these MigratePostImportEvent items as this doesn't seem sustainable over several hundred migrations, even if we start culling out the ones that are done.

Reply:
One idea for how to do this: We could create an interface that we use for all our migration classes that sets an empty `protected function postImport()` that would be called and run from this file. In migration classes where we needed some additional action, we could just add an overridden version of that method.

Reply:
Nice idea--there's some cleanup I'll get to following it, but as far as this post import goes, got it set up this way.